### PR TITLE
Add Translation and Black Hole modal

### PR DESCRIPTION
**Pull request by tin0d0**

### CODE REVIEW

**Review by jileiwang-google — APPROVED** [review]
LGTM

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -91,6 +91,14 @@ function close_comments() {
   document.querySelector('.comments_modal').style.display = 'none';
 }
 
+function open_translate() {
```
Comment (frankpzn) [inline portfolio/src/main/webapp/script.js:94]: Personally I would separate this file by module: put all the translate functions together, put all the comment functions together, etc.

If the file becomes too large, also consider refactor some modules out as their own js file.

**Review by frankpzn — APPROVED** [review]

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (tin0d0) [reply portfolio/src/main/webapp/script.js:94]: Sounds good thank you Frank!

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -135,7 +143,37 @@ function delete_comment(comment) {
...
+  var marker = new google.maps.Marker({position: detroit, map: map});
+}
+
+function requestTranslation() {
+    //grabs the language and text to translate
+  const text = document.getElementById('text').value;
+  const languageCode = document.getElementById('language').value;
+
+  const resultContainer = document.getElementById('result');
+  resultContainer.innerText = 'Loading...';
```
Comment (HidyHan) [inline portfolio/src/main/webapp/script.js:161]: Nit: this should also be in the specified language.

**Review by HidyHan — APPROVED** [review]

**Pull request merged**